### ISSUE TITLE
Table with too low frequency ASVs

### ISSUE THREAD

**Issue by Selucote82** [MaestSi/MetONTIIME#111]
Ciao!

First of all, thank you for developing this tool. It's great to find something that links Nanopore and qiime2 reads.

A problem has arisen after using MetONTIIME regarding ASV frequency, as almost all sequences have been grouped as singletons and only a few ASVs have a maximum of 7 sequences grouped together. The raw reads were joined by barcodes before starting the pipeline and the minIdentity was set to 0.9. What do you think is happening?

Setting the identity to 0.85 or 0.80 might reduce the number of ASVs and increase the frequency of reads per ASV, but I don't know how reliable these results can be.

Thank you!

**Comment by MaestSi** [comment]
Ciao,
I usually keep the **minIdentity** parameter close to 1 and only reduce it in case of memory/time constraints.
Indeed, the clustering of ONT reads, especially with non-ONT specific tools as VSEARCH, may result in reads from the same species being spread over multiple clusters, and reads from different but closely related species being put in the same cluster, just due to the error rate being higher than short-reads. By setting **minIdentity** parameter close to 1, you end up with a number of ASVs very close to the number of filtered reads. So, the concept of ASV is not very useful in this case, but it is needed to generate the proper qza files, i.e. a set of ASV and an associated counts table. ASVs are then aligned to a database of choice, either using Blast or VSEARCH.
Hope this helps,
SM

**Comment by Selucote82** [comment]
Thank you for your quick response!

But if the concept of ASV here is not useful, downstream diversity analyses won't be carried out, because the large amount of singleton will affect to diversity/richness indices...or Am I wrong?

Thank you again!

**Comment by MaestSi** [comment]
What you say is true for phylogeny-based diversity analyses.
If you do non-phylogenetic diversity analyses, as those implemented in **qiime diversity core-metrics** plugin, run by MetONTIIME in **diversityAnalyses** process, those diversity metrics will not require the creation of any phylogenetic tree based on the alignment of ASV. Instead, they will only exploit taxonomy tables collapsed at a specific taxonomic level.
SM

**Comment by Selucote82** [comment]
Ok ok, I get it! Nice to have this "debate". 

Have a nice end of year!

**Comment by MaestSi** [comment]
Same to you!
Best,
SM
